contracts/example/src/contract.rs

```diff
@@ -84,4 +162,47 @@ impl Example {
             &message,
         );
     }
+
+    pub fn send_token(
+        env: &Env,
+        caller: Address,
+        token_id: BytesN<32>,
+        destination_chain: String,
+        destination_address: Bytes,
+        amount: i128,
+        message: Option<Bytes>,
```

Comment:
```suggestion
        recipient: Option<Bytes>,
```
since it needs to be an address that forwards the token